### PR TITLE
Jma/runtime mapping product details

### PR DESCRIPTION
**Pull request by jasonma-shipstation**
add runtime mapping for product details

### CODE REVIEW

File: packages/order-source-runtime/src/mapping/sales-orders-export/output.ts [hunk 1]
```diff
@@ -32,6 +47,7 @@ export function mapItem(item: Output.SalesOrderItem): api.SalesOrderItem {
         isbn: item.product.isbn,
         inventory_id: item.product.inventoryID,
       },
+      details: mapProductDetails(item.product?.identifiers),
```
Comment (jeversmann-shipstation) [inline packages/order-source-runtime/src/mapping/sales-orders-export/output.ts:50]: Shouldn't this be `product.details`?

**Review by jeversmann-shipstation — COMMENTED** [review]
It's kinda weird that we're using `Identifiers` objects for details, but the docs imply that this already works, so 🤷🏽‍♂️ https://connect.shipengine.com/docs/reference/sales-order-item

File: packages/order-source-runtime/src/mapping/sales-orders-export/output.ts (hunk 1 above)
Comment (jasonma-shipstation) [reply packages/order-source-runtime/src/mapping/sales-orders-export/output.ts:50]: ah ur right

**Review by jeversmann-shipstation — APPROVED** [review]

**Comment by jasonma-shipstation** [comment]
release connect

**Comment by jasonma-shipstation** [comment]
increment patch versions

**Pull request merged**